src/highlight.js

```diff
@@ -418,6 +418,9 @@ https://highlightjs.org/
     }
 
     function buildSpan(classname, insideSpan, leaveOpen, noPrefix) {
+      if (!leaveOpen && insideSpan !== '') {
```

Comment:
Wait wouldn't it be `insideSpan === ''`?  What am I missing?

Reply:
I made it wrong, changed it in the generated code, ran the tests and everything worked ;-)